pkg/connection/config.go

```diff
@@ -31,12 +31,26 @@ type Config struct {
 	Token    string `mapstructure:"token"`    // bearer token
 }
 
+// If set, ActiveConfig() returns this configuration.
+// This is intended for use in testing.
+var override *Config
```

Comment:
@theganyo this is not called "config" because that collides with the `config` package name and breaks the call to `config.ActiveName()` below.

Reply:
@seaneganx the only thing above this in the file is the definition of the `Config` type.

Reply:
Would `active` or `cached` be appropriate?

Reply:
+1 for `active`